crates/sdk/src/connector.rs

```diff
@@ -148,5 +148,6 @@ pub trait ConnectorSetup {
         &self,
         configuration: &<Self::Connector as Connector>::Configuration,
         metrics: &mut prometheus::Registry,
+        ndc_models_versions: &Option<versions::SemVer>,
```

Comment:
Can you add a docstring please? I'm not sure exactly which version this is referring to.